server/tables/tables.go

```diff
@@ -427,6 +427,12 @@ func (wf *Workflow) TableName() string {
 	return "Workflows"
 }
 
+type UsageCounts struct {
```

Comment:
FYI this lives in `tables.go` because it will be embedded into the `Usage` table.